packages/spright-components/src/all-components.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Import of all the web components available in Spright AND Nimble.
+ * Production applications are encouraged to import only components
+ * that are required instead of leveraging this file.
+ */
+
+import './rectangle';
+
+import '@ni/nimble-components/all-components';
```

Comment:
Flip the order so all nimble components are registered before spright components.
Nimble's library stuff is in a dist folder

```suggestion
import '@ni/nimble-components/dist/esm/all-components';

import './rectangle';
```